2535.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var differenceOfSum = function (nums) {
+  let eleSum = 0;
+  let digitSum = 0;
+
+  for (let i = 0; i < nums.length; i++) {
+    eleSum += nums[i];
+
+    while (nums[i] > 0) {
+      digitSum += nums[i] % 10;
+      nums[i] = Math.floor(nums[i] / 10);
+    }
+  }
+  
+  let diff = eleSum - digitSum;
```

Comment:
It's good to group variable/constant declarations together.
So you can move this to Line 8 with an initial value 👍

Reply:
FYI, there is a built-in method of `Math` called `Math.abs()`, which helps you to calculate an absolute value.

Reference: [Math.abs()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs)